webpack/assets/javascripts/react_app/API.js

```diff
@@ -17,6 +17,18 @@ export default {
       dataType: 'json',
       data: data,
       error: function (jqXHR, textStatus, errorThrown) {
+        /* eslint-disable no-console */
+        console.log(jqXHR);
+      }
+    });
+  },
+  markGroupNotificationAsRead(group) {
+    $.ajax({
+      url: `/notification_recipients/group/${group}`,
+      contentType: 'application/json',
+      type: 'PUT',
+      error: function (jqXHR, textStatus, errorThrown) {
```

Comment:
Also, I'm not a big fan of logging errors to the console, that works for developers but users won't check there usually. Maybe create a notification for the error itself?